.github/workflows/functional.yml

```diff
@@ -0,0 +1,51 @@
+name: Functional
+
+on:
+  push:
```

Comment:
We don't want it on push? We want it to run in PR, when it is updated. For that, take the events list from BMO.